jest.config.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+process.env.CW_INTERNAL_IPS = '1.0.0.0';
```

Comment:
Should we rename this to `CW_INTERNAL_IP` now that it is a string with a single IP in it? Same question for all occurrences of `.CW_INTERNAL_IP`, or the `cw_internal_ips` secret key